README.md

```diff
@@ -21,17 +21,34 @@
 | 2.12   | [![Maven Central](https://maven-badges.herokuapp.com/maven-central/za.co.absa/abris_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/za.co.absa/abris_2.12) |
 
 ## Supported Spark versions
-On spark 3.0.x and 2.4.x Abris should work without any further requirements.
+On spark 2.4.x, 3.0.x and 3.1.x Abris should work without any further requirements.
 
-On Spark 2.3.x you must declare dependency on ```org.apache.avro:avro:1.8.0``` or higher. (Spark 2.3.x uses Avro 1.7.x so you must overwrite this because ABRiS needs Avro 1.8.0+.)
+On Spark 2.3.x you must declare dependency on ```org.apache.avro:avro:1.8.0``` or higher. 
+(Spark 2.3.x uses Avro 1.7.x so you must overwrite this because ABRiS needs Avro 1.8.0+.)
 
 ## Older Versions
-This is documentation for Abris **version 4**. Documentation for **version 3** is located in 
+This is documentation for Abris **version 5**. Documentation for older versions is located in corresponding branches:
+[branch-4](https://github.com/AbsaOSS/ABRiS/tree/branch-4),
 [branch-3.2](https://github.com/AbsaOSS/ABRiS/tree/branch-3.2).
 
+## Spark Avro Version
+Abris by default uses Spark Avro version 2.4, but it is recommended to use the version matching the used Spark Core version.
+To do this override the dependency on `org.apache.spark:spark-avro`.
+
+For example why is this beneficial: Spark Avro 3.0 accepts nullable columns even for non-nullable avro schema.
+This means the Spark schema doesn't need to be changed (to non-nullable columns) for avro conversion to succeed.
+
+## Confluent Schema Registry Version
+Abris by default uses Confluent client version 5.3.4, but any newer version should work. 
```

Comment:
Can any newer version of Confluent work? I thought the compatibility problem between Confluent v5.4.+ and Spark will only be resolved with Avro 1.10 in Spark 3.2.0

Reply:
From the Abris side it should work, but yes there might be issues elsewhere. I think that it may work if you changed the avro in Spark to the one compatible with schema registry. I am not sure how hard is it to change the libraries that spark is using. 

Maybe I should remove that part and just keep the confluent 5.3 as recommended version until Spark 3.2-Abris is ready.

Reply:
I think the situation has not changed compared to https://github.com/AbsaOSS/ABRiS/issues/165. If you look at the issue there, we figured out that the problem was the Avro version. Since we didn't follow up on a solution to that, I wouldn't add it here.

Reply:
Ok, then I wall remove it from release notes as well, so people don't think that everything will work